src/NetAppFiles/NetAppFiles/help/Get-AzNetAppFilesVault.md

```diff
@@ -129,7 +129,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## OUTPUTS
 
-### Microsoft.Azure.Commands.NetAppFiles.Models.PSNetAppFilesBackupPolicy
```

Comment:
The change of return type is a breaking change. You can create a map from PSNetAppFilesBackupPolicy to PSNetAppFilesVault